invenio_records_files/api.py

```diff
@@ -153,8 +154,11 @@ def __next__(self):
 
     def __contains__(self, key):
         """Test if file exists."""
+        if not isinstance(key, text_type):
```

Comment:
Is this decoding needed? I.e. it's not enough to just remove the ``str()`` call?

Reply:
I was utterly convinced that one needed to have `unicode` objects to use SQLalchemy's `filter`... I was wrong!